source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1611,6 +1612,38 @@ message TransformationPermutePhiOperands {
 
 }
 
+message TransformationPropagateInstructionDown {
+
+  // Propagates an instruction from |block_id| into its successors.
+  // Concretely, the transformation clones the propagated instruction
+  // into some of the successors of |block_id| and removes the original
+  // instruction. Additionally, OpPhi instruction may be added to make sure
```

Comment:
"OpPhi instruction" -> either "an OpPhi instruction" (if just one) or "OpPhi instructions" (if multiple could be required).

Reply:
Reading on, I guess it's "an OpPhi instruction".

Reply:
Fixed in b6fe77b.